components/filter/fields/templates.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Templates
+page_title: FilterField - Templates
+description: Templates for the FilterField
+slug: filter-filterfield-templates
+tags: telerik,blazor,filter,filterfields,templates
+published: True
+position: 10
+---
+
+# FilterField Templates
```

Comment:
For consistency with https://docs.telerik.com/blazor-ui/components/filter/fields/operators

```suggestion
# Filter Field Templates
```